joomla/joomla-cms
# [3.9] script.php. Don't delete this JLayout file! Needed by pr #22107

Issue https://github.com/joomla/joomla-cms/issues/22261

### Summary of Changes
- Remove some deletion lines because https://github.com/joomla/joomla-cms/pull/22107 needs these folders/file

### Testing Instructions
- Whenever you update Joomla 3.9 or use the database fix button the file` /administrator/components/com_languages/layouts/joomla/searchtools/default/bar.php` will be removed and language filter is missing in the BE view `overrides`.

- Apply patch in a 3.9 nightly that contains already https://github.com/joomla/joomla-cms/issues/22261
- Check that the file above exists and was not deleted by earlier actions. That means: You see the language filter in overrides view
- Do a <strike>"Upload&Update" or</strike> (would be stupid)
- Do a database fix that runs script.php
- Check that language filter field is still present
